CHANGELOG.md

```diff
@@ -1,5 +1,57 @@
+ * CodeAction for cvc-complex-type.2.4.b. See [lemminx#1218](https://github.com/eclipse/lemminx/issues/1218).
+ * Support folding for <!DOCTYPE. See [lemminx#1213](https://github.com/eclipse/lemminx/issues/1213).
+ * Added folding setting to keep displaying the closing tag after folding. See [lemminx#1209](https://github.com/eclipse/lemminx/pull/1209).
+ * Provide basic experimental formatter which supports invalid XML. See [lemminx#1195](https://github.com/eclipse/lemminx/issues/1195).
+ * Format with xml:space. See [lemminx#826](https://github.com/eclipse/lemminx/issues/826).
+ * Code action for `SemicolonRequiredInReference`. See [lemminx#665](https://github.com/eclipse/lemminx/issues/665).
+ * Improve XML formatter (option to have Tags in the same line). See [lemminx#594](https://github.com/eclipse/lemminx/issues/594).
+
+### Performance
+
+ * Cancel process of code action. See [lemminx#1220](https://github.com/eclipse/lemminx/issues/1220).
+ * Parse of DOM document should be not done in a Thread. See [lemminx#1216](https://github.com/eclipse/lemminx/pull/1216).
+ * Improve DOM parser memory. See [lemminx#1211](https://github.com/eclipse/lemminx/pull/1211).
+ * Improve XML scanner memory. See [lemminx#1206](https://github.com/eclipse/lemminx/pull/1206).
+ * Wait a bit after change before sending diagnostics. See [lemminx#1162](https://github.com/eclipse/lemminx/issues/1162).
+ * Improve CodeAction performance with CodeAction#data & resolveCodeAction. See [lemminx#941](https://github.com/eclipse/lemminx/issues/941).
+ * Support for `completionItem/resolve`. See [lemminx#616](https://github.com/eclipse/lemminx/issues/616).
+
+### Bug Fixes
+
+ * externalXmlSettings.xmlFileAssociations not used in getXMLSettings. See [#698](https://github.com/redhat-developer/vscode-xml/issues/698).
+ * XSD with enumeration restriction with 1000+ entries fail type assist. See [#679](https://github.com/redhat-developer/vscode-xml/issues/679).
+ * DOCTYPE entities interfering with hover annotation display of tags / attributes. See [lemminx#1212](https://github.com/eclipse/lemminx/pull/1212).
+ * Incorrect error range for `cvc-identity-constraint.4.1`. See [lemminx#1210](https://github.com/eclipse/lemminx/issues/1210).
+ * Resolve uri as system with XML catalog. See [lemminx#1199](https://github.com/eclipse/lemminx/issues/1199).
+ * Preserve invalid content while formatting. See [lemminx#1041](https://github.com/eclipse/lemminx/issues/1041).
+ * Bad element formatting is replaced with <null>. See [lemminx#1034](https://github.com/eclipse/lemminx/issues/1034).
+ * Lone quote inside tag leads to catastrophic formatting. See [lemminx#679](https://github.com/eclipse/lemminx/issues/679).
+ * XML format for bad processing instruction removes following element. See [lemminx#675](https://github.com/eclipse/lemminx/issues/675).
+ * Attribute value without a key is removed while formatting. See [lemminx#305](https://github.com/eclipse/lemminx/issues/305).
```

Comment:
These were fixed by the experimental formatter, so a duplicate of that. I would just leave them out.

Reply:
Or if you feel strongly about including them, just list the bugs at the end of the bug link for the experimental formatter, separated by commas.

Reply:
fixed